RecodoDesktop/RecodoDesktop/MainWindow.xaml.cs

```diff
@@ -1,28 +1,36 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-using System.Windows;
-using System.Windows.Controls;
-using System.Windows.Data;
-using System.Windows.Documents;
+﻿using System.Windows;
 using System.Windows.Input;
-using System.Windows.Media;
-using System.Windows.Media.Imaging;
-using System.Windows.Navigation;
-using System.Windows.Shapes;
 
 namespace RecodoDesktop
 {
-    /// <summary>
-    /// Interaction logic for MainWindow.xaml
-    /// </summary>
     public partial class MainWindow : Window
```

Comment:
This should go to another window, I suppose

Reply:
MainWindow.xaml.cs designed for the logic of working with MainWindow.xaml

Reply:
I mean that the panel you're working on is not a main window of the application, don't you think?

Reply:
It's fine, let it stay as it is for now